.readme-partials.yml

```diff
@@ -1,11 +1,6 @@
 introduction: |-
-  [Google Cloud Logging](https://cloud.google.com/logging/docs) allows you to store, search, analyze,
+  [Stackdriver Logging](https://cloud.google.com/logging/docs) allows you to store, search, analyze,
```

Comment:
do we really need to revert the branding too?

Reply:
perhaps, it's a merge issue. IT should be Google Cloud Logging

Reply:
mm yeah, sry this PR is confusing. Let me close this `revert` attempt. And just create a new PR to directly fix the issue.